x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -7178,10 +7178,8 @@
     "xpack.reporting.errorButton.showReportErrorAriaLabel": "显示报告错误",
     "xpack.reporting.errorButton.unableToFetchReportContentTitle": "无法提取报告内容",
     "xpack.reporting.errorButton.unableToGenerateReportTitle": "无法生成报告",
```

Comment:
Was deleting these intentional?

Reply:
Yes, unfortunately they had to be removed after CI was failing. The strings need to be re-translated so they interpolate the `err` value I'm sending. The removal was done automatically with `node scripts/i18n_check --fix`